framework/wazuh/cluster/dapi/dapi.py

```diff
@@ -389,3 +389,66 @@ def add_request(self, request: bytes):
         """
         self.logger.debug("Received request: {}".format(request))
         self.request_queue.put_nowait(request.decode())
+
+
+class WazuhJSONEncoder(json.JSONEncoder):

```

Comment:
These lines shouldn't be here because this class is defined in wazuh.cluster.common